docs/Berger_POPC_test_case/comp_JMelcrProg/launch.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Launch buildH to reconstruct traj w Hs
+buildH ../start_128popc.pdb -l Berger_POPC -d ../order_parameter_definitions_MODEL_Berger_POPC.def -x ../popc0-25ns_dt1000.xtc -opx popc0-25ns_dt1000_H.xtc
+
+# Rename files
+mv popc0-25ns_dt1000_H.xtc.pdb start_128popc_H.pdb
```

Comment:
"mv **popc0-25ns_dt1000_H.pdb** start_128popc_H.pdb"
(with the above modification)

Reply:
Done